doc/info/tips.3

```diff
@@ -440,19 +512,47 @@ File: TIPS,  Node: Create new bootable ITS, Previous: Copy user directories from
 
    d$nits
 
-6. Start ITS
+6. If you are using a PiDP-10, you must write-protect the disk headers.  To
+   do that, open the SIMH console as described above.  Type the following
+   commands:
+
+   ^E
+   set dpa noheaders
+   continue
+ 
+7. Start ITS
 
    g$
 
-7.  If ITS doesn't start, reenter DSKDMP:
+8.  If ITS doesn't start, reenter DSKDMP:
 
    $U
 
-8. And load and start the new ITS:
+9. And load and start the new ITS:
 
    nits
    $g
 
+10. On some systems, the nits file will not be on the correct disk pack.
+    It is critically important to get this right to avoid an unbootable
+    system.  For example, on PiDP-10 the file must reside on drive 0, pack 2.
+
+    Login to ITS, and type the following:
+
+    :cwd .
+    ^F
+
+    Find the entry for the file named @ its; this is the old copy of the
+    system kernel.  Note the first number on that line; this is the pack
+    number.  For PiDP-10, this will be a 2.  Now find the line for the file
+    named @ nits.  If it is on the same pack as the @ its file, then you can
+    skip the next step.  If it is on a different pack then we must move it,
+    using the following command:
+
+    :move @ nits, dsk0:
+
+    Again use ^F and verify that the pack number is now correct.
+
```

Comment:
I will usually assemble the new ITS directly to DSK0: which writes it to the DSKDMP default mounted pack.  However, sometimes it's moved to another pack by DMPCPY.  In that case I'll use L$n$ to mount the pack in DSKDMP, and after ITS is booted, move the file as you describe.